tests/trogdor/bin/trogdor-local-run.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
It would be good if we could add the ability to read JSON from a file to the Java code, rather than hacking it up in the shell.  The reason is because the shell has some issues with escaping special characters.  Another reason is because this would be useful more generally in all the other trogdor command line commands that take JSON arguments.

So we could just write `./bin/trogdor.sh exec --spec @/my/spec/path`

And then we could also write something like `bin/trogdor.sh client -create-task @/my/task/path`, etc.

I think argparse4j has built-in support for this:
https://argparse4j.github.io/apidocs/net/sourceforge/argparse4j/ArgumentParserBuilder.html#fromFilePrefix-java.lang.String-

Reply:
Sure.

Reply:
fromFilePrefix does not help here because what it does is to read arguments from a file. So you have to put "--spce {....}" into a file, then call ./bin/trogdor.sh exec @/my/spec/path.

Reply:
@cmccabe Updated, please have a look. It reads the SPEC file if the arguments starting with `@`.